packages/fuel-gauge/src/doc-examples.test.ts

```diff
@@ -344,7 +342,7 @@ describe('Doc Examples', () => {
 
   it('can create a predicate', async () => {
     // #region predicate-basic
```

Comment:
The `predicate-basic` context isn't used in the docs.
- Suggest to add reference in the docs to `predicate-basic`